Framework/API/test/RemoteJobManagerFactoryTest.h

```diff
@@ -149,16 +149,7 @@ class RemoteJobManagerFactoryTest : public CxxTest::TestSuite {
     const Mantid::Kernel::FacilityInfo &prevFac = Mantid::Kernel::ConfigService::Instance().getFacility();
 
     Mantid::Kernel::ConfigService::Instance().setFacility("SNS");
-
-    // These two create should throw a NotFoundError because the
-    // RemoteJobManager classes are missing and have not done a
-    // DECLARE_REMOTEJOBMANAGER. A positive test is done in the tests
-    // of the job managers
-    TSM_ASSERT_THROWS("create() with " + FermiName + " should throw because its job manager is not declared",
-                      Mantid::API::IRemoteJobManager_sptr jobManager =
-                          Mantid::API::RemoteJobManagerFactory::Instance().create("Fermi"),
-                      const Mantid::Kernel::Exception::NotFoundError &);
-
+    TS_ASSERT(Mantid::API::RemoteJobManagerFactory::Instance().create("Fermi"));
```

Comment:
I'm not clear why this test has changed?

Reply:
Previously this test failed because it (incorrectly) relied on the Mantid Framework not loading in correctly
This is because the linker will strip out unused shared objects if we don't explicitly statically init them. I've got a workaround in the PR that fixes this for all cases, which means that alg actually runs